master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
```

Comment:
I meant rather than say a "Node entry" I'd use the resource terminology - so "Every Calico node has an associated Node resource and various sub configuration (Workload endpoint resources,  Host endpoint resources, IP address assignments, CIDR affinity blocks, low level configuration)"